packages/utils/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './type'
```

Comment:
why you're moving type/prototype.ts? that file is hacky and I don't want new code to use them without appropriate reason